html-forms/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <div>Email:
```

Comment:
There is different HTML tag we should be using in order to label inputs (please fix all the labels in this file)